runtime/gc_ctrl.c

```diff
@@ -156,40 +156,40 @@ static uintnat norm_custom_min (uintnat p)
 
 CAMLprim value caml_gc_set(value v)
 {
-  uintnat newpf;
-  uintnat newminwsz;
-  uintnat new_custom_maj, new_custom_min, new_custom_sz;
-  CAML_EV_BEGIN(EV_EXPLICIT_GC_SET);
+  uintnat newminwsz = caml_norm_minor_heap_size (Long_val (Field (v, 0)));
+  uintnat newpf = norm_pfree (Long_val (Field (v, 2)));
```

Comment:
Naive question: why was the documentation for the `major_heap_increment` not documented to reflect the fact that the parameter is not currently used anywhere in the runtime?

Reply:
I have no idea. My guess: it's work and no one has done it yet. Note that there are many things that would require more work in the GC statistics (we have to rework the system to understand which stats to make domain-local and which ones to report globally, the computation of the maxima is wrong, etc.), so the best thing we can say for 5.0 is that it is a Work in Progress.